content/influxdb/cloud-serverless/guides/prototype-evaluation.md

```diff
@@ -96,15 +99,14 @@ InfluxDB Cloud Serverless and Cloud Dedicated support SQL and InfluxQL.
 | SQL      | Natively supported        | Natively supported       |
 | InfluxQL | Natively supported        | Natively supported       |
 
-The v2 API (which uses the Flux language for querying) is reachable in InfluxDB
+The v2 Query API (which uses the Flux language for querying) is reachable in InfluxDB
 Cloud Serverless, but isn't supported. If you plan to use InfluxDB Cloud
 Serverless as an evaluation or staging platform for InfluxDB Cloud Dedicated,
 use SQL or InfluxQL.
 
 ### API and client library differences
 
-Because the v2 Query API uses Flux, customers should avoid using the v2 API when
-querying InfluxDB Cloud Serverless as an evaluation, staging, or prototyping
+Because the v2 Query API uses Flux, customers should avoid using the v2 Query API and its associated tooling when querying InfluxDB Cloud Serverless as an evaluation, staging, or prototyping
```

Comment:
```suggestion
Because the v2 query API uses Flux, customers should avoid using the v2 query API and its associated tooling when querying InfluxDB Cloud Serverless as an evaluation, staging, or prototyping
```